gpt4all-chat/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added
 - Add Qwen2-1.5B-Instruct to models3.json (by [@ThiloteE](https://github.com/ThiloteE) in [#2759](https://github.com/nomic-ai/gpt4all/pull/2759))
+- Enable translation feature for seven languages: English, Spanish, Italian, Chinese Simplified, Chinese Traditional, Romanian (by [@manyoso](https://github.com/manyoso) in [#2833](https://github.com/nomic-ai/gpt4all/pull/2833))
```

Comment:
To save on typing, I haven't been giving attribution for PRs made by you or me:
```suggestion
- Enable translation feature for seven languages: English, Spanish, Italian, Chinese Simplified, Chinese Traditional, Romanian ([#2833](https://github.com/nomic-ai/gpt4all/pull/2833))
```
Also, you say seven but that's only six. I think you missed Portuguese.